mqtt-impl/src/main/java/io/streamnative/pulsar/handlers/mqtt/MQTTServerConfiguration.java

```diff
@@ -97,6 +97,13 @@
     )
     private String defaultNamespace = "default";
 
+    @FieldContext(
+            category = CATEGORY_MQTT,
+            required = false,
+            doc = "Default Pulsar topic persistent that the MQTT server used."
+    )
+    private boolean defaultTopicPersistent = true;
```

Comment:
It should be defaultTopicDomain = persistent, if users want to change to non-persistent, just change the defaultTopicDomain = non-persistent